src/opm/parser/eclipse/share/keywords/000_Eclipse100/W/WCONINJE

```diff
@@ -3,10 +3,10 @@
     {"name" : "TYPE"         , "value_type" : "STRING" },
     {"name" : "STATUS"       , "value_type" : "STRING" , "default" : "OPEN"},
     {"name" : "CMODE"        , "value_type" : "STRING" },
-    {"name" : "RATE" , "value_type" : "DOUBLE" , "dimension" : "ContextDependent"},
-    {"name" : "RESV" , "value_type" : "DOUBLE" , "dimension" : "ReservoirVolume/Time"},
-    {"name" : "BHP"  , "value_type" : "DOUBLE" , "dimension" : "Pressure" , "default" : 6895},
-    {"name" : "THP"  , "value_type" : "DOUBLE" , "dimension" : "Pressure"},
+    {"name" : "RATE" , "value_type" : "UDA" , "dimension" : "ContextDependent"},
+    {"name" : "RESV" , "value_type" : "UDA" , "dimension" : "ReservoirVolume/Time"},
+    {"name" : "BHP"  , "value_type" : "UDA" , "dimension" : "Pressure" , "default" : 6895},
```

Comment:
Not strictly speaking part of this change-set, but this line caught my eye.  6895 looks a lot like 100,000 psi i bars.  Is this default value safe for all unit systems?